components/cluster/command/exec.go

```diff
@@ -14,27 +14,12 @@
 package command
 
 import (
-	"errors"
-
-	"github.com/fatih/color"
-	"github.com/joomcode/errorx"
-	perrs "github.com/pingcap/errors"
-	"github.com/pingcap/tiup/pkg/cluster/spec"
-	"github.com/pingcap/tiup/pkg/cluster/task"
-	"github.com/pingcap/tiup/pkg/logger"
-	"github.com/pingcap/tiup/pkg/logger/log"
-	"github.com/pingcap/tiup/pkg/meta"
-	"github.com/pingcap/tiup/pkg/set"
+	"github.com/pingcap/tiup/pkg/cluster/deploy"
```

Comment:
I think `deploy` is not a proper package name, how about using something like `cmd` or `subcmd` or just leave them under `pkg/cluster`?